twindle-cli/pdf/index.js

```diff
@@ -5,7 +5,7 @@ const { createPdf } = require("./create-pdf");
 
 async function generatePDF(tweets, outputPath) {
   // creates the html content
-  const htmlContent = await renderTemplate({ thread: tweets }, "Thread");
+  const htmlContent = await renderTemplate({ thread: tweets.data, common: tweets.common }, "Thread");
```

Comment:
Did you also changed the function calling this function, because collectTweets didnt return something called **common** before ?